tasks/build-scan-push/build-scan-push.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  creationTimestamp: null
+  name: build-scan-push
+spec:
+  stepTemplate:
+    env:
+    - name: HOME
+      value: /tekton/home
+    envFrom:
+    - secretRef:
+        name: jx-boot-job-env-vars
+        optional: true
+    name: ""
+    resources: {}
+    workingDir: /workspace/source
+  steps:
+  - image: gcr.io/kaniko-project/executor:v1.6.0-debug
+    name: build-container
+    resources: {}
+    script: |
+      #!/busybox/sh
+      source .jx/variables.sh
+      cp /tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson /kaniko/.docker/config.json
+      /kaniko/executor $KANIKO_FLAGS --context=/workspace/source --dockerfile=${DOCKERFILE_PATH:-Dockerfile} --destination=$PUSH_CONTAINER_REGISTRY/$DOCKER_REGISTRY_ORG/$APP_NAME:$VERSION --no-push --tarPath image.tar
+  - image: ghcr.io/alexkondrat/trivydb:latest
```

Comment:
tag instead of latest

Reply:
latest has to  be done for trivy (as you want the latest vulns in the database)